src/app/authentication/extend-sessions.ts

```diff
@@ -80,3 +80,25 @@ export const extendSessions = async (
       countOfInactiveSessions === 0 ? "none" : countOfInactiveSessions,
   })
 }
+
+export const maybeExtendSession = async ({
+  sessionId,
+  expiresAt,
+  monthsToExtend = 12,
+}: {
+  sessionId: string
+  expiresAt: string
+  monthsToExtend?: number
+}): Promise<boolean | void | Error> => {
+  const futureDate = new Date()
+  // related to session.lifespan value in kratos.yaml
+  futureDate.setMonth(futureDate.getMonth() + monthsToExtend)
```

Comment:
How is Month represented?
Does this work when we are crossing a year?
1 (Jan) + 12 = 13 (Jan - 1 year later)?

Reply:
I would not say that the interface `setMonth` can mutate the year is intuitive. But fair enough.